lib/Twig/Cache/Filesystem.php

```diff
@@ -73,6 +78,15 @@ public function write($key, $content)
         if (false !== @file_put_contents($tmpFile, $content) && @rename($tmpFile, $key)) {
             @chmod($key, 0666 & ~umask());
 
+            if (self::FORCE_BYTECODE_INVALIDATION == ($this->options & self::FORCE_BYTECODE_INVALIDATION)) {
+                // Compile cached file into bytecode cache
+                if (function_exists('opcache_invalidate') && ini_get('opcache.enable')) {
+                    opcache_invalidate($key);
```

Comment:
I think the second parameter (force) should be set to true as we know the file changed and otherwise the option FORCE_BYTECODE_INVALIDATION would be misleading.